zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala

```diff
@@ -149,6 +149,23 @@ class InMemorySpanStore extends SpanStore with CollectAnnotationQueries {
       .toList
   }
 
+  override protected def getTraceIdsByDuration(
+    serviceName: String,
```

Comment:
PS I may be totally wrong on interpreting how the queries work in relation
to service name. If SpanStoreSpec tests don't answer these questions, we
need to make them.